sandy/sections/mf2.py

```diff
@@ -0,0 +1,107 @@
+import sandy
+
+__author__ = "Luca Fiorito"
+__all__ = [
+        "read_mf2",
+        "write_mf2"
+        ]
+
+
+allowed_mt = ( 151 )
+
+mf = 2
+mt=151
+
+def read_mf2(tape, mat):
+   
+    if mt in allowed_mt:
```

Comment:
This `if/else` shall be removed as it will never raise the error. It could be here if you pass `mt` as a kwarg, which is not needed here.